modules/ROOT/pages/apid-behavioral-headers.adoc

```diff
@@ -102,6 +102,17 @@ MS2-Status-Code:: Specifies the status code or codes that the mocking service ca
 If you use this header together with MS2-Error-Rate, you must specify both a non-error status code and an error status code. Separate the two status codes with a comma. For example, you might want the mocking service to return a 201 message, if there is no error, or a 401 status code, if there is an error; in this case, you would specify `201,401` for the value.
```

Comment:
Hey Robert, I was viewing the "transform" of this file and just happened to notice that there is a missing period at the end of the first para.

Reply:
Note also that you should remove "the" before "API Console":


Official name | API Console | Products and components are like names, there is no "the" in front of them.
-- | -- | --